src/App.jsx

```diff
@@ -1,24 +1,17 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
 import './App.css'
+import React from 'react';
+import TodoList from './TodoList1';
+import AddTodoForm from './AddTodoForm1';
```

Comment:
You may have introduced typos to your imports. There are no TodoList1.jsx or AddTodoForm1.jsx files.

Since this PR is already merged, you can make corrections while working on your next lesson. Please make sure that you run your app before opening a PR - Vite would have given you error messages about those files missing.